crates/ruff/src/rules/flake8_pyi/rules/long_string_or_bytes_in_stub.rs

```diff
@@ -0,0 +1,43 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use rustpython_parser::ast::{self, Constant, Expr, Ranged};
+
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+
+#[violation]
+pub struct LongStringOrBytesInStub;
+
+/// ## What it does
+/// Checks for `str` or `bytes` literals longer than 50 characters in stubs.
+///
+/// ## Why is this bad?
+/// Extremely long strings used as argument defaults or otherwise are unlikely to be useful for
+/// users. Consider replacing them with ellipses.
```

Comment:
Questionable reasoning, but my archaeology in `flake8-pyi` didn't turn up any better reasons.

Reply:
Here's an explanation: https://github.com/PyCQA/flake8-pyi/blob/ceab86d16b822d12abae1d8087850d0bc66d2f52/CHANGELOG.md?plain=1#LL110C1-L115C23

Reply:
Funny, good catch.

Reply:
@charliermarsh I improved the comment and added an example.